### ISSUE TITLE
You can't hide channels/categories.

### ISSUE THREAD

**Issue by Maximal98** [uowuo/abaddon#45]
You can't hide channels. in the official client you would do this by muting them and then setting it to hide muted channels. i find this approch a little stupid because i think it would be nicer to have the ability to mute OR hide channels.

**Comment by yusdacra** [comment]
I agree with this, though I'd like if the option of "hide channels/categories that are muted" was kept since I personally make use of it.
